lib/menubar/utils.lua

```diff
@@ -226,21 +214,23 @@ function utils.lookup_icon_uncached(icon_file)
         return false
     end
 
-    if icon_file:sub(1, 1) == '/' and is_format_supported(icon_file) then
+    local icon_file_ext = icon_file:match(".*%.(.-)$")
```

Comment:
i see the changes proposed by @blueyed to make it faster are not incorporated yet: https://github.com/awesomeWM/awesome/pull/2468#issuecomment-437753145

Reply:
His changes are included...

original: local icon_file_ext = icon_file:gsub(".*%.","")
his: local icon_file_ext = icon_file:match("^.+(%..+)$")
final: local icon_file_ext = icon_file:match(".*%.(.-)$")

why it changed...

gsub  ->  match
:  definitely faster [thanks @blueyed]

match("^.+(%..+)$")  ->  match(".*%.(.-)$")
```
icon_file = "/home/user/.config/awesome/theme/name/.icons/file.ext"
print("1>",icon_file:match("^.+(%..+)$") )
print("2>",icon_file:match(".*%.(.-)$") )
icon_file = "/home/user/.config/awesome/theme/name/.icons/file."
print("3>",icon_file:match("^.+(%..+)$") )
print("4>",icon_file:match(".*%.(.-)$") )
1>	.ext
2>	ext
3>	.icons/file.
4>	
```
:  almost identical but "b>" is what is needed

Reply:
ok, thanks for the explanation